bluemira/utilities/opt_tools.py

```diff
@@ -92,7 +93,7 @@ def regularised_lsq_fom(x, A, b, gamma):
         Residual vector (Ax - b)
     """
     residual = np.dot(A, x) - b
-    number_of_targets = float(len(residual))
+    number_of_targets = floatify(len(residual))
```

Comment:
Does len not return an int? Is floatify needed here?

Reply:
No it doesnt AFAIK